### ISSUE TITLE
ansible not working in a multi-user environment

### ISSUE THREAD

**Issue by styv** [Homebrew/homebrew-core#6975]
If I install ansible (2.2.0.0) on my machine (let’s say I used the user account "adminuser"), I get an error when trying to run ansible from another user account (let’s say "regularuser").

Running `brew test ansible` from the "adminuser" account works fine. However, running `brew test ansible` from the "regularuser" account results in an error:

    ERROR! Unexpected Exception: [Errno 13] Permission denied: '/usr/local/Cellar/ansible/2.2.0.0_1/libexec/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/PKG-INFO'

This is seems to be a file permissions issue:

    $ ls -l /usr/local/Cellar/ansible/2.2.0.0_1/libexec/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info
    total 40
    -rw-------  1 adminuser  admin  710  1 Nov 04:43 PKG-INFO
    -rw-------  1 adminuser  admin  230  1 Nov 04:43 SOURCES.txt
    -rw-------  1 adminuser  admin    1  1 Nov 04:43 dependency_links.txt
    -rw-r--r--  1 adminuser  admin   93  1 Nov 04:43 installed-files.txt
    -rw-------  1 adminuser  admin   12  1 Nov 04:43 top_level.txt

After issuing `chmod a+r /usr/local/Cellar/ansible/2.2.0.0_1/libexec/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/*` as user "adminuser" the tests run fine from both, "adminuser" and "regularuser" accounts.

**Comment by MikeMcQuaid** [comment]
Does this work fine for any/all other formulae you have installed?

**Comment by styv** [comment]
At least I haven't discovered any other issues so far. We use this setup on our CI server where the brew installation is by an admin user while the CI jobs are run by a regular (non-admin) user. So this works at least for the formulae ant, autoconf, automake, doxygen, mercurial, node4-lts.

Note that for all other python packages installed with ansible, i.e. those found in /usr/local/Cellar/ansible/2.2.0.0_1/libexec/lib/python2.7/site-packages/*.egg-info, the permissions are set to 644 for all files. prettytable is the only exception here.

**Comment by tdsmith** [comment]
This seems like a prettytable packaging error but I think the cleaner ought to catch this.

**Comment by s01ipsist** [comment]
This broke all my ansible provisioning too.

```
sudo chmod a+R /usr/local/Cellar/ansible/2.2.0.0_1/libexec/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/*
```
solved the problem for me.

**Comment by MikeMcQuaid** [comment]
Could someone try and open a pull request? [This document](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged) should help and I'm happy to walk you through anything else.

Thanks!

**Comment by styv** [comment]
Closed by merged pull request #7848.
